contract_api/dao/service_repository.py

```diff
@@ -172,3 +172,20 @@ def curate_service(self, org_id, service_id, curate):
         except:
             self.rollback_transaction()
             raise
+
+    def get_service_media(self,service_id,org_id):
+        query = "SELECT `row_id`, org_id, service_id, url, `order`, file_type, asset_type, alt_text,ipfs_url FROM contract_unittest_db.service_media where service_id = %s and org_id = %s order by `order` "
+        service_media = self.connection.execute(query, (service_id, org_id))
+
+        if len(service_media) > 0:
+            return service_media
+        return None
+
+    def create_service_media(self,org_id,service_id,url,order,file_type,asset_type,alt_text,ipfs_url,service_row_id):
+        query = "INSERT INTO contract_unittest_db.service_media (org_id, service_id, url, `order`, file_type, asset_type, alt_text,ipfs_url,service_row_id,created_on, updated_on) VALUES(%s, %s, %s, %s, %s, %s, %s, %s,%s, %s,%s)"
```

Comment:
Dont have 10 arguments to the function. Create a dict with the appropriate keys. You can pass the dict itself to the execute method